worlds/sm64ex/docs/setup_en.md

```diff
@@ -36,7 +36,7 @@ If it still crashes, recheck if you typed the launch options correctly (Describe
 # Manual Compilation (Linux/Windows)
 
 Dependencies for Linux: `sdl2 glew cmake python make`.
-Dependencies for Windows: `mingw-w64-x86_64-gcc mingw-w64-x86_64-glew mingw-w64-x86_64-SDL2 git make python3 mingw-w64-x86_64-cmake`
+Dependencies for Windows: `mingw-w64-x86_64-gcc mingw-w64-x86_64-glew mingw-w64-x86_64-SDL2 git make python3 cmake`
```

Comment:
is this content change accidental?

Reply:
No. It didn't appear when I did my merge update. Will revert when I get home.